synfig-studio/src/gui/docks/dock_layergroups.cpp

```diff
@@ -67,6 +69,10 @@ Dock_LayerGroups::Dock_LayerGroups():
 	action_group_group_ops(Gtk::ActionGroup::create("action_group_dock_layergroups")),
 	group_action_manager(new GroupActionManager)
 {
+	// Make Sets toolbar buttons small for space efficiency
+	auto context = get_style_context();
```

Comment:
Since the context is not used further, it can be simplified (just a suggestion):
```
get_style_context()->add_class("synfigstudio-efficient-workspace");
```

Reply:
Sounds legit... Tried it and encountered this
![Screenshot from 2020-10-23 06-28-42](https://user-images.githubusercontent.com/24214509/96944661-0e0bd200-14f9-11eb-8045-9194da78e7ee.png)
Maybe we're suppose to wait for the object to be returned and assigned to the variable first...

Reply:
Why are you assigning the (void) returned by add_class to a variable? :)